### PR TITLE
Adding initial Hebrew support for sentences.

### PR DESCRIPTION
**Pull request by haim-b**

### CODE REVIEW

File: sentences/he/homeassistant_HassTurnOn.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: "he"
+intents:
+  HassTurnOn:
+    data:
+      - sentences:
+          - "הדלק [את] [ה]<name>"
+          - "הדלקה [של] [ה]<name>"
+          - "להדליק [את] [ה]<name>"
```
Comment (yuvalabou) [inline sentences/he/homeassistant_HassTurnOn.yaml:8]:
```suggestion
          - "להדליק [את] [ה]<name>"
          - "הפעל [את] [ה]<name>"
          - "הפעלה [של] [ה]<name>"
          - "תפעיל [את] [ה]<name>"
```

**Comment by synesthesiam** [comment]
@haim-b Can you please remove the "HassToggle" intent? We decided it wasn't worth including for now.

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ See the [documentation](docs/README.md) for more information.
 | `dk` | Danish           | (position open)    |
 | `en` | English          | (position open)    |
 | `fr` | French           | @benjaminlecouteux |
-| `he` | Hebrew           | @leranp            |
+| `he` | Hebrew           | @haim-b, @leranp   |
```
Comment (yuvalabou) [inline README.md:28]:
```suggestion
| `he` | Hebrew           | @haim-b, @leranp, @yuvalabou    |
```

File: CODEOWNERS [hunk 3]
```diff
@@ -10,6 +10,9 @@ sentences/fr/ @benjaminlecouteux
 responses/fr/ @benjaminlecouteux
 tests/fr/ @benjaminlecouteux
 
+sentences/he/ @haim-b
+responses/he/ @haim-b
+tests/he/ @haim-b
 sentences/he/ @leranp
 responses/he/ @leranp
 tests/he/ @leranp
```
Comment (thecode) [inline CODEOWNERS:18]:
```suggestion
sentences/he/ @leranp @haim-b
responses/he/ @leranp @haim-b
tests/he/ @leranp @haim-b
```

File: README.md (hunk 2 above)
Comment (thecode) [inline README.md:28]:
```suggestion
| `he` | Hebrew           | @leranp, @haim-b |
```

**Review by thecode — CHANGES_REQUESTED** [review]
codeowners should be on the same line and it is better to do them in order of adding (reduce line changes_

**Review by thecode — APPROVED** [review]
Merging this into a branch so others can help with the tests

**Pull request merged**